galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/TsqException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.cicsts;
+
+public class TsqException extends TsqManagerException {
```

Comment:
Can we define what type of errors will be reporting using this type of exception ? and how it differs from the TsqManagerException parent ?

Reply:
TsqManagerException happens for errors in TSQ manager provisioning and TsqException happens for errors in the TSQ manager methods. The same is updated in the comments.

Hi @techcobweb , FYI.  Similar comments are missing in almost all Manager exceptions and exceptions I checked in in Galasa source.  For example : CECI, CEMT, CEDA, ZOS Manager, Zos batch manager, Zos file manager etc.